nRF700x driver doxygen docs (#11372)
nrfconnect/sdk-nrf: doc/nrf/drivers/nrf700x/low_api_radio_test.rst

Radio Test mode
---------------

This mode is used for testing the RF performance of the nRF700X device, mainly to characterize the RF.

> Note: Documentation for data structures definitions is disabled for this module, will be enabled in the future.

Reviewer: ```suggestion
.. note::
   Documentation for data structure definitions is disabled for this module.
   It will be enabled later on.
```
